### PR TITLE
add easyblock for EggLib

### PR DESCRIPTION
**Pull request by boegel**
requires https://github.com/hpcugent/easybuild-easyblocks/pull/810

### CODE REVIEW

**Comment by boegel** [comment]
@wpoely86: please review

File: easybuild/easyblocks/e/egglib.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+            env.setvar(varname, '%s:%s' % (os.path.join(self.installdir, subdir), os.environ.get(varname, '')))
+
+        # build/install Python package
+        py_subdir = os.path.join(self.builddir, 'egglib-py-%s' % self.version)
+        try:
+            os.chdir(py_subdir)
+        except OSError, err:
+            raise EasyBuildError("Failed to move to: %s", )
+
+        super(EB_EggLib, self).build_step()
```
Comment (wpoely86) [inline easybuild/easyblocks/e/egglib.py]: use `PythonPackage.` With multiple inheritance, it's much more readable.

File: easybuild/easyblocks/e/egglib.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        self.cfg.update('installopts', "--install-lib %s" % os.path.join(self.installdir, self.pylibdir))
+        self.cfg.update('installopts', "--install-scripts %s" % os.path.join(self.installdir, 'bin'))
+
+        super(EB_EggLib, self).install_step()
+
+    def sanity_check_step(self):
+        """Custom sanity check for EggLib."""
+        custom_paths = {
+            'files': ['bin/egglib', 'lib/libegglib-cpp.a'],
+            'dirs': ['include/egglib-cpp', 'lib/python2.7/site-packages'],
```
Comment (wpoely86) [inline easybuild/easyblocks/e/egglib.py]: hard coded `2.7`?

**Comment by boegel** [comment]
Reworked and retested, thanks for the review @wpoely86!

**Pull request merged**